packages/labs/polyfills/scoped-registry.js

```diff
@@ -0,0 +1,330 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+const NativeHTMLElement = window.HTMLElement;
+const nativeDefine = window.customElements.define;
+const nativeGet = window.customElements.get;
+const nativeRegistry = window.customElements;
+
+const definitionForElement = new WeakMap();
+const pendingRegistryForElement = new WeakMap();
+const globalDefinitionForConstructor = new WeakMap();
+// TBD: This part of the spec proposal is unclear:
+// > Another option for looking up registries is to store an element's
+// > originating registry with the element. The Chrome DOM team was concerned
+// > about the small additional memory overhead on all elements. Looking up the
+// > root avoids this.
+const scopeForElement = new WeakMap();
+
+// Constructable CE registry class, which uses the native CE registry to
+// register stand-in elements that can delegate out to CE classes registered
+// in scoped registries
+window.CustomElementRegistry = class {
```

Comment:
Shouldn't this be part of the web components polyfills? Is there something Lit specific about this polyfill?

Reply:
Yes. I just added it for testing purposes as right now I can't retrieve it through npm

Reply:
I changed to retrieve the package from github. It can be changed later to retrieve it from npm once published